packages/govuk-frontend/src/govuk/components/accordion/accessibility.puppeteer.test.mjs

```diff
@@ -2,13 +2,22 @@ import { axe, render } from '@govuk-frontend/helpers/puppeteer'
 import { getExamples } from '@govuk-frontend/lib/components'
 
 describe('/components/accordion', () => {
+  let axeRules
+
+  beforeAll(() => {
+    axeRules = {
+      // Ignore colour contrast (enhanced) from WCAG Level AAA
```

Comment:
Thanks for adding these @domoscargin

Shall we be nice to our future selves and say what it was ignored for?

```suggestion
      /**
       * Ignore 'Element has insufficient color contrast' for WCAG Level AAA
       * 
       * Affects 'Show all sections', 'Show' and 'Hide' toggle links
       */
```

Reply:
Done!